website/src/all.cpp

```diff
@@ -81,8 +81,6 @@ namespace XXX {
 #include "C_IStream_read.cpp"
 #include "C_IStream_seekg.cpp"
 #include "C_IStream_tellg.cpp"
-#include "gamma.cpp"
```

Comment:
If `gamma.cpp` and `makePreviewImage.cpp` have been subsumed by `previewImageExamples.cpp`, they should be removed from the git repo, via 
`git rm gamma.cpp makePreviewImage.cpp`

Reply:
One question before pushing new changes - does it make sense to use `Imath::clamp()` instead of `std::clamp()`? `clamp` is only available in std from c++17.